html/Mogo/js/slideshow/message-slider/message-slider.js

```diff
@@ -0,0 +1,34 @@
+import {components} from './message-components.js';
+import {slide} from '../helper-functions.js';
+
+export function runMessageSlider() {
+  components.then((response) => {
+    const {
+      messageSlides,
+      messageNextBtn,
+      messagePrevBtn,
+    } = response;
+
+    let messageCurrentSlide = 0;
+
+    slide(messageSlides, 0);
+
+    messageNextBtn.addEventListener('click', () => {
+      if (messageCurrentSlide === messageSlides.length - 1) {
```

Comment:
think how to calc next messageCurrentSlide with modulo operator

Reply:
The same for `peopleCurrentSlide`. Btw, try to reduce code duplication — message slider and people slider almost the same